Cargo.toml

```diff
@@ -61,6 +61,7 @@ async-std-runtime = ["async-std", "async-tungstenite/async-std-runtime"]
 tokio-runtime = ["tokio", "async-tungstenite/tokio-runtime"]
 fetcher = []
 bytes = ["dep:bytes"]
+serde0 = []
```

Comment:
I added serde0 as feature because there were config attributes checking for feature named "serde0" but it was not defined and clippy was complaining about it

Reply:
https://blog.rust-lang.org/2024/07/25/Rust-1.80.0.html#checked-cfg-names-and-values